src/rules/pls_no_land.rs

```diff
@@ -41,6 +46,11 @@ fn pls_no_land_impl(path: &PathBuf) -> anyhow::Result<Vec<diagnostic::Diagnostic
         return Ok(vec![]);
     }
 
+    // Filter out well known files that should have the word donotland in them (like toolbox.toml)
+    if path.file_name().map_or(false, |f| f == "toolbox.toml") {
+        return Ok(vec![]);
+    }
+
```

Comment:
nit, optional: might make sense to make a function, similar to `is_binary_file` to make this a little more self-descriptive